src/Phinx/Db/Adapter/PdoAdapter.php

```diff
@@ -425,11 +444,13 @@ public function createSchemaTable()
                 $table->addColumn('version', 'biginteger', array('limit' => 14))
                       ->addColumn('start_time', 'timestamp', array('default' => 'CURRENT_TIMESTAMP'))
                       ->addColumn('end_time', 'timestamp', array('default' => 'CURRENT_TIMESTAMP'))
+                      ->addColumn('migration_name', 'string', array('limit' => 100))
```

Comment:
I'd like to see this column placed after `version` instead of at the end of the table.